cypress/e2e/5TextFontCustomisation.cy.js

```diff
@@ -0,0 +1,13 @@
+describe("Text Font Customisation", () => {
+  it("Customising the font", () => {
+    cy.visit("/");
+    cy.contains("Change Font").click();
+
+    // Select the Roboto font from the dropdown box
+    cy.get(".rounded.absolute.text-gray-700.pt-1").contains("Roboto");
```

Comment:
Could we assert that once clicking on this, that the text font updates to `Roboto`?

Reply:
The assertion may not take place as there is no text content in that text box, so it may not be possible to assert that but I'll try it out nonetheless

Reply:
@gracebilliris 

I can't assert it as there is no text content inside the text box

`cy.get("#inputted-text").should("have.css", "font-family", `"Open Sans"`);
`
Tried to check it like this, but you can't assert it as there is no content inside

```
<failure message="Timed out retrying after 4000ms: Expected to find content: &apos;&quot;Open Sans&quot;&apos; within the element: &lt;ul.rounded.absolute.hidden.text-gray-700.pt-1.group-hover:block.w-56&gt; but never did." type="AssertionError"><![CDATA[AssertionError: Timed out retrying after 4000ms: Expected to find content: '"Open Sans"' within the element: <ul.rounded.absolute.hidden.text-gray-700.pt-1.group-hover:block.w-56> but never did.
    at Context.eval (webpack://sis-team-24/./cypress/e2e/5TextFontCustomisation.cy.js:9:51)]]></failure>
```

Reply:
Could you type text in the box and then assert on it?